tests/run-pass/linux-getrandom.rs

```diff
@@ -0,0 +1,17 @@
+// Unfortunately, compiletest_rs does not support 'only-linux',
+// so we need to ignore Windows and OS X instead
```

Comment:
```suggestion
// so we need to ignore Windows and macOS instead.
```
It's a sentence, it should end in a full stop. (Also applies to some other comments in this PR.)

We have historically not been consistent about this in Miri, but we are trying to be consistent for new PRs.